subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesDependenciesIntegrationTest.groovy

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testing.testsuites
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+
+
+
+class TestSuitesDependenciesIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
Can you also add a few tests here to demonstrate some other differences:
- "test" test suite has dependencies on the project and user created ones do not 
- Like above, but add a dependency on the project for the other test suite
- Exercise implementation, runtimeOnly and compileOnly dependencies for a test suite
- Add dependencies through the top-level dependencies block with a user created test suite

Reply:
I'm cranking through these scenarios; look for an update soon.

Reply:
PTAL - I have added coverage for these cases.